spec/support/vcr_cassettes/lighthouse/benefits_discovery/200_response_with_all_params.yml

```diff
@@ -0,0 +1,77 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sandbox.lighthouse.va.gov/benefits-discovery-service/v0/recommendations
```

Comment:
Please mask the URI in [support/vcr](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/vcr.rb)

Reply:
@RachalCassity I'm not sure I understand this. I don't recall ever having to do this for any other cassettes. Looking around the vets-api, I see a mix of this approach and bare urls. Is there a reason some are masked and others not? And does the entire uri need to be masked? Looking at examples, it seems like maybe only the host matters and not the path.

Reply:
I would mask the hostname. You can mask the entire URI if all cassettes involved use the same path.

Reply:
Thanks for explaining. First time I've heard of the rule. I'll fix it shortly.